tests/test_hypothesis_plots.py

```diff
@@ -172,3 +178,43 @@ def test_plot_ridge(datatree, combined, plot_kwargs, labels_shade_label):
                 assert all(key in child for child in pc.viz.children.values())
         elif key not in ("remove_axis", "ticklabels"):
             assert all(key in child for child in pc.viz.children.values())
+
+
+# plot_ppc tests
+ppc_kind_value = st.sampled_from(("kde", "cumulative"))
+ppc_group = st.sampled_from(("prior", "posterior"))
+
+
+@given(
+    plot_kwargs=st.fixed_dictionaries(
+        {},
+        optional={
+            "kind": plot_kwargs_value,
+            "predictive": plot_kwargs_value,
+            "observed": plot_kwargs_value,
+            "aggregate": plot_kwargs_value,
+            "observed_rug": plot_kwargs_value,
+            "title": plot_kwargs_value,
+            "remove_axis": st.just(False),
+        },
+    ),
+    kind=ppc_kind_value,
+    group=ppc_group,
```

Comment:
All relevant plot specific arguments should be here, especially if they affect what gets plotted or how the data is processed _within the function_. Thus, `observed` and `aggregate` (toggling `observed_rug` on too if `observed` is true), `facet_dims` and `num_pp_samples` should be here.